src/lib/CRC/crc.cpp

```diff
@@ -0,0 +1,38 @@
+#include "crc.h"
```

Comment:
Generic. Please harmonize datatypes and use either main types or fixed width type (recommended).